### ISSUE TITLE
Should not lint on ... in function signature

### ISSUE THREAD

**Issue by radbasa** [Appsilon/box.linters#21]
## box.linters version

0.0.0.9006

## Source code to lint

```R
some_function <- function(x, y, ...) {
  sum(x, y, ...)
}

some_function(1, 2)

another_function <- function(x, y, ...) {
  sum(x, y)
}

another_function(1, 2)
```

## Lint command

```R
lintr::lint(code, linters = lintr::linters_with_defaults(defaults = box.linters::rhino_default_linters))
```

## Expected result

Should not lint on `...` in function signature and use in function.
